DependencyInjection/CmfSeoExtension.php

```diff
@@ -81,8 +81,17 @@ public function load(array $configs, ContainerBuilder $container)
             $this->loadSonataAdmin($config['sonata_admin_extension'], $loader, $container, $sonataBundles);
         }
 
-        if ($this->isConfigEnabled($container, $config['alternate_locale'])) {
-            $this->loadAlternateLocaleProvider($config['alternate_locale'], $container);
+        if ($config['content_listener']) {
```

Comment:
Yeah, I thought we did things with the service that was configured as content listener. I agree with the tagging.

+1 for a booleanNode now